src/balena.ts

```diff
@@ -1,19 +1,26 @@
+import { fileURLToPath } from 'node:url';
 import {
 	aliasTable,
 	generateAbstractSqlModel,
 	renameVarResourcesName,
 	optimizeSchema,
-} from './abstract-sql-utils';
+} from './abstract-sql-utils.js';
 
-import * as userHasDirectAccessToApplication from './features/applications/models/user__has_direct_access_to__application';
-import * as deviceAdditions from './features/devices/models/device-additions';
-import * as releaseAdditions from './features/ci-cd/models/release-additions';
+import * as userHasDirectAccessToApplication from './features/applications/models/user__has_direct_access_to__application.js';
+import * as deviceAdditions from './features/devices/models/device-additions.js';
+import * as releaseAdditions from './features/ci-cd/models/release-additions.js';
 
-export const apiRoot = 'resin';
+export const apiRoot = 'resin.js';
```

Comment:
Isn't this going to change the version in the OData `/resin/<resource>` endpoints as well?